extractor/src/main/java/rs/valence/extractor/extractors/Effects.java

```diff
@@ -0,0 +1,55 @@
+package rs.valence.extractor.extractors;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import net.minecraft.registry.Registries;
+import rs.valence.extractor.Main;
+
+public class Effects implements Main.Extractor {
+    public Effects() {
+    }
+
+    @Override
+    public String fileName() {
+        return "effects.json";
+    }
+
+    public static String toPascalCase(String str) {
```

Comment:
nit: doesn't need to be public

We should probably just put this in another class. I imagine a `Utils` or `ValenceUtils` class would be a good place for this.

Reply:
Alright. I moved it to a `ValenceUtils` class